ext/doc/Cassandra/Bigint.php

```diff
@@ -21,103 +21,134 @@
 /**
  * A PHP representation of the CQL `bigint` datatype
  */
-final class Bigint implements Value, Numeric {
-
+final class Bigint implements \Stringable, Value, Numeric
```

Comment:
Stringable is PHP8 only

Reply:
same

Reply:
Yeah but you have to update composer.json